### PR TITLE
feat: add i18n ally vscode settings

### PR DESCRIPTION
**Pull request by shunkakinoki**
Purpose: Add custom settings for the i18n-ally extension (https://github.com/antfu/i18n-ally)
 
I discovered this extension a couple of days ago, and it drastically improves the developer experience when having to deal with i18n and translations. It also features editing & commenting for i18n all managed with vscode and git! I really encourage you to try it out if you already haven't already

btw, 
sorry if it the extension and the settings are already utilized by the contributors, 
and I may need time to contribute to the source code itself since I haven't come across developing with vue before :)

### CODE REVIEW

**Comment by shunkakinoki** [comment]
Below is the demo gif:

2020-06-20 at 0 47 02 - Sapphire Lion

**Comment by shunkakinoki** [comment]
The source code inline translations demo:

2020-06-20 at 0 49 54 - Plum Fox

File: .vscode/settings.json [hunk 1]
```diff
@@ -6,5 +6,9 @@
   "editor.tabSize": 2,
   "editor.formatOnType": true,
   "editor.formatOnPaste": true,
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "i18n-ally.localesPaths": [
+    "lang",
+    "functions/src/lang"
```
Comment (isamu) [inline .vscode/settings.json:12]: Locale files are in  "lang" directory.

Comment (shunkakinoki) [reply .vscode/settings.json]: Definitely! my bad

**Comment by isamu** [comment]
Thank you!

File: .vscode/settings.json (hunk 1 above)
Comment (isamu) [reply .vscode/settings.json]: Oh, sorry. There was a lang directory above. I overlooked it.
You can also add this setting. 

Thanks.

**Pull request merged**